packages/template-hello-world-ng/src/app/item/items.component.html

```diff
@@ -26,8 +26,8 @@
 <GridLayout>
     <ListView [items]="items">
         <ng-template let-item="item">
-            <StackLayout>
-                <Label [nsRouterLink]="['/item', item.id]" [text]="item.name"></Label>
+            <StackLayout >
+                <Label [text]="item.name"  [nsRouterLink]="['/item', item.id]" backgroundColor="red"></Label>
```

Comment:
`backgroundColor="red"` was probably a debugging helper? This file seems like needing no changes -- as a best practice I tend to skim over all the files after I submit a PR to clean up such leftovers.